mlflow/tracking/client.py

```diff
@@ -875,7 +875,7 @@ def _upload_ended_trace_info(
 
     def _exclude_immutable_tags(self, tags: Dict[str, str]) -> Dict[str, str]:
         """Exclude immutable tags e.g. "mlflow.user" from the given tags."""
-        return {k: v for k, v in tags.items() if not k.startswith("mlflow.")}
```

Comment:
The `mlflow.sourceRun` tag is generated by client side so we can't reject all tags starts with `mlflow.`.

Reply:
Isn't `mlflow.sourceRun` immutable too ?

Reply:
Nope, they are mutable indeed (search for `userImmutableRunTags` in universe).